truss/remote/baseten/remote.py

```diff
@@ -267,7 +286,7 @@ def patch(
                 patch_request.prev_hash == patch_request.next_hash
                 or len(patch_request.patch_ops) == 0
             ):
-                logger.info("No changes observed, skipping deployment")
+                logger.info("No changes observed, skipping patching")
```

Comment:
`rich.print` is probably more user-friendly than `logger.info`; users don't need to know which modules the logs are coming from and we can color code

Reply:
go for it.